server/src/models/category.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose';
+
+const categorySchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    image: {
+      type: String,
+    }
+  },
+  {
+    timestamps: true,
+  }
+);
+
+const Category = mongoose.model('category', categorySchema);
```

Comment:
use all the string as string constants, so that you only have to change one file.

Reply:
I am not clear about this point. Could you explain it briefly?

Reply:
Is this about the the model names ( for example `'category'` used in mongoose.model ?

Reply:
yeah i was talking about 'category' only to move this into a string constant file.

Reply:
[Is this](https://github.com/pesto-students/batch-10-tripen/blob/b5748a170f836c47240e8dd7a01620ce6d25f9b5/server/src/models/user.js#L71) a good approach?